code/style.css

```diff
@@ -1,7 +1,500 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+ body {
+   margin: 0;
+   font-family: 'Source Sans Pro', sans-serif;
+   background-color: white;
+   color: #333;
+ }
 
-.active + .answer {
-  /* some styling here */
-}
+ a {
+   color: #333;
+ }
+
+ /***************** Hero Video ****************/
+
+ .hero-wrapper-container {
+   display: flex;
+   justify-content: center;
+   justify-self: stretch;
+   align-items: flex-start;
+ }
+
+ .hero-styling {
+   width: 100%;
+   height: 100%;
+   position: relative;
+   object-fit: cover;
+   overflow: hidden;
+   left: 0;
+   right: 0;
+   top: 0;
+   bottom: 0;
+   text-align: center;
+   background: black;
+ }
+
+ .hero-inner-container {
+   display: flex;
+   flex: 1;
+ }
+
+ .video {
+   flex: 1;
+ }
+
+ .hero-video {
+   opacity: 0.8;
+ }
+
+ .title-container {
+   position: absolute;
+ }
+
+ .title h1 {
+   display: flex;
+   font-size: 3rem;
+   font-weight: 600;
+   margin-top: 9rem;
+   color: #fff;
+   letter-spacing: 0.2rem;
+   margin-bottom: 0;
+ }
+
+ /* Title styles for mobile and tablets */
+
+ @media only screen and (max-width: 767px) {
+   .title h1 {
+     margin-top: 7rem;
+     font-size: 2rem;
+   }
+ }
+
+ @media only screen and (max-width: 991px) and (min-width: 768px) {
+   .title h1 {
+     font-size: 2.5rem;
+   }
+ }
+
+ .title p {
+   width: 400px;
+   padding: 2rem;
+   margin: auto;
+   color: #fff;
+   font-size: 2rem;
+   font-weight: 500;
+ }
+
+ /* paragrah styles for mobile version */
+
+ @media only screen and (max-width: 767px) {
+   .title p {
+     font-size: 1.5rem;
+   }
+ }
+
+ /* Hero button */
+
+ .btn {
+   width: 200px;
+   border: 3px solid white;
+   padding: 1.2rem;
+   margin: 20px auto 0;
+   font-size: 1.5rem;
+   font-weight: 500;
+   background-color: rgba(255, 255, 255, 0.158);
+   color: #fff;
+ }
+
+ .btn:hover {
+   cursor: url('pointer.png'), pointer;
+   transform: scale(1.2);
+   transition: 0.1s;
+   background-color: rgba(255, 255, 255, 0.336);
+ }
+
+ @media only screen and (max-width: 767px) {
+   .btn {
+     font-size: 1rem;
+     padding: 0.5rem;
+     margin: 10px auto 0;
+     width: 150px;
+   }
+ }
+
+ .info-section-wrapper {
+   display: flex;
+   flex-direction: row;
+   align-items: stretch;
+   justify-content: center;
+   margin-top: 50px;
+ }
+
+ /* Mobile view for form and acardion / making them jump into a column */
+
+ @media only screen and (max-width: 767px) {
+   .info-section-wrapper {
+     flex-direction: column;
+     margin-top: 10px;
+   }
```

Comment:
Great use of flexbox. 
Could I suggest to use `flex-wrap: wrap` instead of adding a media query for the jump. If you add the flex-wrap outside of the media query the form and the accordion will jump to column from row when the space is not enough.

Reply:
cool suggestion thank you , I will try it out